packages/core-components/src/components/DismissableBanner/DismissableBanner.tsx

```diff
@@ -102,23 +102,36 @@ export const DismissableBanner = (props: Props) => {
   const classes = useStyles();
   const storageApi = useApi(storageApiRef);
   const notificationsStore = storageApi.forBucket('notifications');
-  const rawDismissedBanners =
-    notificationsStore.snapshot<string[]>('dismissedBanners').value ?? [];
+  const dismissedBannersSnapshot =
+    notificationsStore.snapshot<string[]>('dismissedBanners');
```

Comment:
I believe that the right way of consuming the storage API is in one single place via `useObservable`: https://github.com/backstage/backstage/blob/7612daf25a6756da6f09db377584fb8fe8032f69/plugins/home/src/components/CustomHomepage/CustomHomepageGrid.tsx#L107

Reply:
Good catch, thank you!